gnowsys-ndf/gnowsys_ndf/es_initialize.py

```diff
@@ -0,0 +1,751 @@
+from gnowsys_ndf.ndf.models import *
+from elasticsearch import Elasticsearch
+from bson import json_util
+import os
+import sys
+import json
```

Comment:
Convention:
- Always keep direct `import` of main/python-modules on top followed by package/project import.